File: Difficulty: Easy/Palindrome String/palindrome-string.cpp
class Solution {
  public:
    // Function to check if a string is a palindrome.
    bool isPalindrome(string& s) {
        // code here
        stack<char> st;
        for(int i=0; i<s.size();i++){
            st.push(s[i]);
        }
        int i=0;
        while(!st.empty() && i<s.size()){
            if (st.top()!=s[i]) return false;
            st.pop();
            i++;
        }
        return true;
        
    }
};

File: Difficulty: Easy/Palindrome String/palindrome-string.py
class Solution:
    def pal(self,s,l):
        if l>=len(s)//2:
            return True
        if s[l]!=s[len(s)-l-1]:
            return False
     
        return self.pal(s,l+1)
 
    def isPalindrome(self, s):
        # code here
        return self.pal(s,0)
        
        
